src/fights.ts

```diff
@@ -1423,6 +1423,10 @@ const freeFightSources = [
         withChoice(1498, 1, () => use($item`closed-circuit pay phone`));
       }
 
+      if (have($item`Rufus's shadow lodestone`)) {
+        adv1(bestShadowRift(), -1, "");
```

Comment:
are we just assuming the user never does the forest items choice? imo it probably would be worth doing once a day but it's fine if we say "nah" since mafia doesn't do a good job handling "3 if present otherwise 2"

Reply:
we handle the forest choice elsewhere